components/post_view/post_attachment_opengraph/post_attachment_opengraph.jsx

```diff
@@ -248,8 +248,8 @@ export default class PostAttachmentOpenGraph extends React.PureComponent {
         });
     }
 
-    isRemovePreview(post) {
-        if (post && post.props) {
+    isRemovePreview(post, currentUser) {
```

Comment:
This doesn't look related to this change. It should be reverted if that's the case.